jina/serve/runtimes/request_handlers/worker_request_handler.py

```diff
@@ -292,6 +291,7 @@ async def handle(
             docs_matrix=WorkerRequestHandler.get_docs_matrix_from_request(
                 requests,
                 field='docs',
+                as_list=not self.args.docs_by_executor
```

Comment:
Why do we change the argument naming internally? can't we use `docs_by_executor` everywhere instead of `as_list`?